airflow/executors/mesos_executor.py

```diff
@@ -0,0 +1,475 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import threading
+from builtins import str
+from queue import Queue
+from typing import Any, Optional
+
+from avmesos.client import MesosClient
+
+from airflow import configuration
+from airflow.exceptions import AirflowException
+from airflow.executors.base_executor import BaseExecutor, CommandType
+from airflow.models.taskinstance import TaskInstance, TaskInstanceKey
+from airflow.settings import Session
+from airflow.utils.state import State
+
+DEFAULT_FRAMEWORK_NAME = 'Airflow'
+FRAMEWORK_CONNID_PREFIX = 'mesos_framework_'
+
+
+def get_framework_name():
+    """Get the mesos framework name if its set in airflow.cfg"""
+    if not configuration.conf.get('mesos', 'FRAMEWORK_NAME'):
+        return DEFAULT_FRAMEWORK_NAME
+    return configuration.conf.get('mesos', 'FRAMEWORK_NAME')
```

Comment:
```suggestion
    return configuration.conf.get('mesos', 'FRAMEWORK_NAME', fallback=DEFAULT_FRAMEWORK_NAME)
```
Btw. we usually import `conf` instead of `configuration`, this make the code simpler to read 😉

Reply:
ok, thanks @turbaszek  I will change it. :-)